Library/Koala/KoalaTests.swift

```diff
@@ -46,6 +46,31 @@ final class KoalaTests: TestCase {
     XCTAssertEqual("GB", properties?["user_country"] as? String)
   }
 
+  func testDefaultPropertiesVoiceOver() {
+    let client = MockTrackingClient()
+    let koala = Koala(client: client)
+
+    var isVoiceOverRunning = { true }
+
+    withEnvironment(isVoiceOverRunning: isVoiceOverRunning) {
+      koala.trackAppOpen()
+
+      let properties = client.properties.last
+
+      XCTAssertTrue(properties?["is_voiceover_running"] as! Bool)
```

Comment:
The force cast necessary?

Reply:
I always considered force unwrapping in tests an OK approach since tests are not shipping to production and you actually really want the value to be what you tell it to be so if it's not you have a problem.

Anyways, I'm fine with using optionals...I've also removed any other force unwraps in this test suite for consistency. It looks like `XCTAssertEqual` is a better API to use when dealing with optionals since `XCTAssertTrue`/`False` expects non-optionals.

Reply:
Thanks, yeah I think it's better for tests to fail rather than potentially crash. I didn't mean for you to go and change all of the other force unwraps but thanks for doing that! 😄 I was just thinking that the force _cast_ in this instance maybe wasn't necessary 👍